docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -487,22 +503,67 @@ export default function AppLayoutDocsFooter(props) {
                   {t('submit')}
                 </Button>
               </DialogActions>
-              {rating !== 1 && (
-                <Typography id="feedback-description" color="text.secondary">
-                  {t('feedbackMessageToGitHub.usecases')}
-                  <br />
-                  {t('feedbackMessageToGitHub.callToAction.text')}
+            </Box>
+          </form>
+        </Collapse>
+        <Divider sx={{ mb: 4 }} />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column-reverse', md: 'row' },
+            justifyContent: 'space-between',
+            gap: { xs: 4, lg: 12 },
+          }}
+        >
+          <Box flexGrow={1}>
+            <Typography variant="body2" fontWeight="bold">
+              Keep up to date
+            </Typography>
+            <Typography variant="body2" color="text.secondary" sx={{ mb: 2 }}>
+              Join our newsletter for regular updates. No spam ever.
+            </Typography>
+            <EmailSubscribe fullWidth sx={{ mb: 1 }} />
+          </Box>
+          <Divider sx={{ display: { xs: 'block', sm: 'none' } }} />
+          <Stack direction="column" spacing={3}>
+            <Typography variant="body2" fontWeight="bold">
+              Engage further with us
```

Comment:
How about this for the header?

```suggestion
              Connect with us
```

Reply:
Sweet, I like that!